src/sum.py

```diff
@@ -1,2 +1,8 @@
-def sum(a: int, b: int) -> int:
+def sum(a:int, b:int) -> int:
     return a + b
+
+print("Insert the first number:")
+a =  input() 
+print("Insert the second number:")
+b = input()
```

Comment:
se noti queste righe sono molto simili tra loro

prova a mettere la funzionalità di richiesta ed inserimento del numero in una funzione

inoltre, metti il codice di esecuzione dentro if __name__ == "__main__" come qui https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do